tests/integration_test/chat_service_test.py

```
@@ -3849,6 +4007,56 @@
     )
 
 
+@responses.activate
+def test_get_instagram_user_posts(monkeypatch):
+    from kairon.shared.channels.instagram.processor import InstagramProcessor
+
+    access_token = chat_client_config["config"]["headers"]["authorization"][
```

Ruff (0.12.2)

Local variable `access_token` is assigned to but never used

Remove assignment to unused variable `access_token`

(F841)